lib/utils.js

```diff
@@ -131,21 +131,46 @@ utils.promisify = function(fn) {
   })
 }
 
-utils.wrightDataDirectory = (function() {
-  const ubuntuOnWindows = (process.env.PATH || '').match(/:\/mnt\/c\/users\/([a-z0-9-_]+)/i) // eslint-disable-line
-  return process.platform === 'win32'
-    ? path.join(process.env.HOMEPATH || '', 'wright') // eslint-disable-line
-    : ubuntuOnWindows
-      ? ('C:\\Users\\' + ubuntuOnWindows[1] + '\\.wright')
-      : path.join(process.env.HOME || '', '.wright') // eslint-disable-line
-}())
-
-utils.getAppDataDirectory = function(name) {
-  const wrightPath = utils.wrightDataDirectory
-      , projectPath = path.join(wrightPath, name)
+utils.wrightDataDirectory = function(process) {
+  const home = process.env.HOMEPATH || process.env.HOME || ''
+
+  const wslPath =
+    [process.env]
+    .filter( x => x.WSL_DISTRO_NAME )
+    .flatMap( x => x.PATH ? [x.PATH] : [] )
```

Comment:
I think full flatMap is node +12. Currently wright works with node 8 and up (maybe even 6), but maybe it's ok if 2.0 only supports 12 and up considering developers usually run the latest version locally? (then again I also change node versions now and then to test things). Polyfill flatMap perhaps ? :P

Reply:
I always forget this 😪

Yeah easy to drop before release 👍